lib/components/Modal.js

```diff
@@ -7,12 +7,15 @@ var elementClass = require('element-class');
 var renderSubtreeIntoContainer = require("react-dom").unstable_renderSubtreeIntoContainer;
 
 var SafeHTMLElement = ExecutionEnvironment.canUseDOM ? window.HTMLElement : {};
+var AppElement = document.body;
```

Comment:
Seems there's a comment on every PR about something breaking when rendered on the server. Perhaps it's time for a simple unit test to catch these issues.  The dev community is pretty equally split on those that write isomorphic apps and those who don't, as such we can't expect everyone that works on this to know that what they're doing is going to break something.